static/app/components/preprod/preprodBuildsDistributionTable.tsx

```diff
@@ -0,0 +1,120 @@
+import type {ReactNode} from 'react';
+import {Fragment} from 'react';
+import styled from '@emotion/styled';
+
+import {Text} from 'sentry/components/core/text';
+import {Tooltip} from 'sentry/components/core/tooltip';
+import {SimpleTable} from 'sentry/components/tables/simpleTable';
+import {t} from 'sentry/locale';
+import {formatNumberWithDynamicDecimalPoints} from 'sentry/utils/number/formatNumberWithDynamicDecimalPoints';
+import type {BuildDetailsApiResponse} from 'sentry/views/preprod/types/buildDetailsTypes';
+
+import {
+  FullRowLink,
+  PreprodBuildsCommonHeaderCells,
+  PreprodBuildsCommonRowCells,
+  PreprodBuildsCreatedHeaderCell,
+  PreprodBuildsCreatedRowCell,
+} from './preprodBuildsTableCommon';
+
+interface PreprodBuildsDistributionTableProps {
+  builds: BuildDetailsApiResponse[];
+  organizationSlug: string;
+  showProjectColumn: boolean;
+  content?: ReactNode;
+  onRowClick?: (build: BuildDetailsApiResponse) => void;
+}
+
+function DownloadCountHeaderCell() {
+  return <SimpleTable.HeaderCell>{t('Download Count')}</SimpleTable.HeaderCell>;
+}
+
+function DownloadCountRowCell({build}: {build: BuildDetailsApiResponse}) {
+  const downloadCount = build.distribution_info?.download_count ?? 0;
+  return (
+    <SimpleTable.RowCell>
+      <Text>{formatNumberWithDynamicDecimalPoints(downloadCount, 0)}</Text>
+    </SimpleTable.RowCell>
+  );
+}
+
+export function PreprodBuildsDistributionTable({
+  builds,
+  content,
+  onRowClick,
+  organizationSlug,
+  showProjectColumn,
+}: PreprodBuildsDistributionTableProps) {
+  const rows = builds.map(build => {
+    const linkUrl = `/organizations/${organizationSlug}/preprod/${build.project_id}/${build.id}/install/`;
+    const isInstallable = build.distribution_info?.is_installable ?? false;
+    const isRowDisabled = !isInstallable;
+    const rowContent = (
+      <Fragment>
+        <PreprodBuildsCommonRowCells
+          build={build}
+          showInteraction={!isRowDisabled}
+          showProjectColumn={showProjectColumn}
+        />
+        <DownloadCountRowCell build={build} />
+        <PreprodBuildsCreatedRowCell build={build} />
+      </Fragment>
+    );
+
+    const RowComponent = isRowDisabled ? DisabledRow : SimpleTable.Row;
+
+    return (
+      <RowComponent key={build.id} variant={isRowDisabled ? 'faded' : 'default'}>
+        {isRowDisabled ? (
+          <Tooltip title={t('Not Installable')} containerDisplayMode="contents">
+            <DisabledRowContent>{rowContent}</DisabledRowContent>
+          </Tooltip>
+        ) : (
+          <FullRowLink to={linkUrl} onClick={() => onRowClick?.(build)}>
+            {rowContent}
+          </FullRowLink>
+        )}
+      </RowComponent>
+    );
+  });
+
+  return (
+    <BuildsDistributionTable showProjectColumn={showProjectColumn}>
+      <SimpleTable.Header>
+        <PreprodBuildsCommonHeaderCells showProjectColumn={showProjectColumn} />
+        <DownloadCountHeaderCell />
```

Comment:
Similar to `DownloadCountRowCell`, would recommend just inlining the `DownloadCountHeaderCell` content. No need for separate components for those since they're used once